pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file defining the project's dependencies:
+
+`composer init`
+
+- Create a new project from a given version of an existing package:
+
+`composer create-project {{project_name}} {{/path/to/my/project}} {{version}}`
+
+- Install the packages specified in composer.json file in the current directory:
```

Comment:
Yes you are right. I have simplified the description, removing the directory part.